command/ca/admin/add.go

```diff
@@ -0,0 +1,99 @@
+package admin
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	mgmtAPI "github.com/smallstep/certificates/authority/mgmt/api"
+	"github.com/smallstep/certificates/linkedca"
+	"github.com/smallstep/cli/errs"
+	"github.com/smallstep/cli/flags"
+	"github.com/smallstep/cli/utils/cautils"
+	"github.com/urfave/cli"
+)
+
+func addCommand() cli.Command {
+	return cli.Command{
+		Name:      "add",
+		Action:    cli.ActionFunc(addAction),
+		Usage:     "add an admin to the CA configuration",
+		UsageText: `**step ca admin add** <provisioner> <subject> [**--super**]`,
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "super",
+				Usage: `Give administrator SuperAdmin privileges.`,
+			},
+			flags.X5cCert,
+			flags.X5cKey,
+			flags.PasswordFile,
+			flags.CaURL,
+			flags.Root,
+		},
+		Description: `**step ca admin add** adds an admin to the CA configuration.
+
+## POSITIONAL ARGUMENTS
+
+<provisioner>
+: The name of the provisioner
+
+<subject>
+: The subject name that must appear in the identifying credential of the admin.
+
+## EXAMPLES
+
+Add regular Admin:
+'''
+$ step ca admin add admin-jwk max@smallstep.com admin-jwk
+'''
+
+Add SuperAdmin:
+'''
+$ step ca admin add admin-jwk max@smallstep.com --super
+'''
+`,
+	}
+}
+
+func addAction(ctx *cli.Context) (err error) {
+	if err := errs.NumberOfArguments(ctx, 2); err != nil {
+		return err
+	}
+
+	args := ctx.Args()
+	provName := args.Get(0)
+	subject := args.Get(1)
+
+	typ := linkedca.Admin_ADMIN
+	if ctx.IsSet("super") {
```

Comment:
Use `ctx.Bool("super")`, because this is valid `--super=false`, and I suppose that condition would be true.